cmd/meroxa/root/environments/create.go

```diff
@@ -105,19 +105,25 @@ func (c *Create) setUserValues(e *meroxa.CreateEnvironmentInput) {
 	}
 }
 
+func stringSliceToMap(input []string) map[string]interface{} {
```

Comment:
This should be extracted in some point as part of builder, once we change how config flags are used in other commands, but I think it's fine we leave it here for now.